.github/workflows/examples.yaml

```diff
@@ -63,6 +63,12 @@ jobs:
             pipenv run yarn algob run scripts/transfer/master-fund-john.js
             pipenv run yarn algob run scripts/transfer/tesla-to-john.js
 
+      - name: Example bond token
+      working-directory: ./examples/bond
+      run: |
+        pipenv run yarn algob deploy
+        pipenv run yarn algob run scripts/run/run.js
```

Comment:
shouldn't we run tests as well?

Reply:
yes, we are already running tests from `main.yaml`.